src/Troopers/BehatContexts/Context/ExtendedEntityContext.php

```diff
@@ -39,4 +40,88 @@ public function beforeScenario($event)
             }
         }
     }
+
+    /**
+     * @Then /^I should find (\d+) (.*) like:$/
+     *
+     * @param $nbr
+     * @param $name
+     * @param TableNode $table
+     *
+     * @throws \Exception
+     */
+    public function existObjectLikeFollowing($nbr, $name, TableNode $table)
+    {
+        $entityName = $this->resolveEntity($name)->getName();
```

Comment:
The code from `$entityName` to `$this->getQueryParams($entityName, $headers, $row);` is duplicated, can you please factorize it in a method?